docs/framework-guides/docs/framework/advances-configuration.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Advanced Configuration
+sidebar_label: ⚙️ Advanced Configuration
+sidebar_position: 3
+---
+
+# ⚙️ Advanced configuration
+The Ocean framework is based on a Python HTTP client called [HTTPX](https://www.python-httpx.org/). The HTTPX client provides out-of-the-box configuration capabilites, which allows configuring the client to work with:
```

Comment:
Add a line break between this paragraph and the list underneath it, just for markdown readability